routes/_middleware.ts

```diff
@@ -86,7 +88,19 @@ export const handler = async (
     newHeaders.set("Cache-Control", "no-cache, no-store, private");
   }
 
-  const newResponse = new Response(initialResponse.body, {
+  if (
+    url.searchParams.has("asRaw") &&
+    newHeaders.get("content-type")?.includes("text/html")
+  ) {
+    const html = await initialResponse.text();
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(html, "text/html");
+    const firstChild = doc?.querySelector("body")?.firstElementChild;
+    firstChild?.querySelector("script")?.remove();
+    bodyReadable = firstChild?.outerHTML ?? null;
+  }
```

Comment:
nit: You can do it later.

I'd rather prefer put this on [fresh handler](https://github.com/deco-cx/live.ts/blob/main/handlers/fresh.ts#L24) (together along `asJson`) just for simplicity and localization.

PS: There's no overhead in copying headers twice